railwayroutegenerator/routegenerator.py

```diff
@@ -28,9 +28,6 @@ def traverse_edge(self, edge, direction, current_route=None, active_signal=None)
                 # And start the next route from this signal
                 active_signal = signal
                 current_route = Route(signal)
-            else:
```

Comment:
@arneboockmeyer this was resulting ValueErrors being thrown as there are two Ausfahr_Signals on the same edge in Potsdam Hbf, but for different directions. removing this else resolves the problem and results in correct routes being found as we only respect the signals for the appropriate direction.